packages/context/src/context-view.ts

```diff
@@ -155,17 +160,54 @@ export class ContextView<T = unknown> extends EventEmitter
 }
 
 /**
- * Create a context view as a getter
+ * Create a context view as a getter with the given filter
+ * @param ctx Context object
+ * @param bindingFilter A function to match bindings
+ * @param session Resolution session
+ */
+export function createViewGetter<T = unknown>(
+  ctx: Context,
+  bindingFilter: BindingFilter,
+  session?: ResolutionSession,
+): Getter<T[]>;
+
+/**
+ * Create a context view as a getter with the given filter and sort matched
+ * bindings by the comparator.
  * @param ctx Context object
  * @param bindingFilter A function to match bindings
+ * @param bindingComparator A function to compare two bindings
  * @param session Resolution session
  */
 export function createViewGetter<T = unknown>(
   ctx: Context,
   bindingFilter: BindingFilter,
+  bindingComparator?: BindingComparator,
+  session?: ResolutionSession,
+): Getter<T[]>;
+
+/**
+ * Create a context view as a getter
+ * @param ctx Context object
+ * @param bindingFilter A function to match bindings
+ * @param bindingComparatorOrSession A function to sort matched bindings or
+ * resolution session if the comparator is not needed
+ * @param session Resolution session if the comparator is provided
+ */
```

Comment:
I think the actual implementation should not have any tsdoc comments. We have already described the two flavors recognized by the compiler when validating places calling this method. WDYT?

Reply:
The comments are for maintainers only. I assume strong-docs or tsdocs only expose the method signatures (not implementation).